aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AppVersionHelper.cs

```diff
@@ -13,7 +13,7 @@ public class AppVersionHelper
         /// Gets current version of the application.
         /// It's also shown in the web page.
         /// </summary>
-        public const string Version = "4.3.0.0";
+        public const string Version = "4.5.0.0";
```

Comment:
@maliming should this versioning follow the release?

I don't think we follow abp version for template

https://github.com/aspnetboilerplate/module-zero-core-template/releases

Reply:
I am not very sure, @ismcagdas What do you think?

Reply:
No, it is not same with ABP version, it is the version of the template but it should be 4.6 now. It seems like v4.6 of this template has been relesed (https://github.com/aspnetboilerplate/module-zero-core-template/releases) but the version is not upgraded. And the milestone seems wrong as well. I will fix it all.